drivers/scheduler/k8s/k8s.go

```diff
@@ -2169,8 +2169,12 @@ func (k *K8s) ValidateVolumeSnapshotRestore(ctx *scheduler.Context, timeStart ti
 		if err != nil {
 			return fmt.Errorf("snapshot: %s is not complete. %v", snapshotData.Metadata.Name, err)
 		}
+		snapID := snapshotData.Spec.PortworxSnapshot.SnapshotID
+		if snapshotData.Spec.PortworxSnapshot.SnapshotType == snap_v1.PortworxSnapshotTypeCloud {
+			snapID = "in-place-restore-" + vol
+		}
 		// validate each snap restore
-		if err := driver.ValidateVolumeSnapshotRestore(vol, snapshotData.Spec.PortworxSnapshot.SnapshotID, timeStart); err != nil {
+		if err := driver.ValidateVolumeSnapshotRestore(vol, snapID, timeStart); err != nil {
```

Comment:
Pass in `snapshotData` instead of `snapID` here. This file shouldn't know if it was a Portworx volume

Reply:
moved to `volume/portworx`